application/event_processor/dynamodb.py

```diff
@@ -0,0 +1,48 @@
+import boto3
+from json import dumps, loads
+import hashlib
+from decimal import Decimal
+from typing import Any, Dict
+from boto3.dynamodb.types import TypeSerializer
+from time import time
+from os import environ
+from aws_lambda_powertools.logging.logger import Logger
+
+TTL = 157680000  # int((365*5)*24*60*60) . 5 years in seconds
+logger = Logger(child=True)
+
+
```

Comment:
Can this file be removed to the common folder so it can be shared with the new fifo-dlq-handler.

Reply:
Sure. It makes sense